Bugzilla/Component.pm

```diff
@@ -374,6 +376,17 @@ sub bug_count {
   return $self->{'bug_count'};
 }
 
+# Lazy-load the bug_description_template column
+sub bug_description_template {
+  my $self = shift;
+  if (!exists $self->{'bug_description_template'}) {
```

Comment:
If you add a `try { } catch {};` here (and a `use Try::Tiny;` to the top of the module) you could avoid having to do a separate DB code change push

Reply:
Okay 👍

Reply:
Looks like `try-catch` isn't required here. Can retrieve 2 templates at once.